tests/test_core/test_models.py

```
@@ -0,0 +1,260 @@
+
+def test_processing_settings_validation():
+    """Test ProcessingSettings validation."""
+    settings = ProcessingSettings(
+        batch_id_format="%Y%m%d_%H%M%S",
+        cache_downloads=True,
+        cache_by_year=True
+    )
+    assert settings.batch_id_format == "%Y%m%d_%H%M%S"
+    
+    # Test batch ID format
+    batch_id = datetime.now().strftime(settings.batch_id_format)
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)